modules/react/popup/lib/Popup.tsx

```diff
@@ -18,16 +18,97 @@ export interface PopupProps {
   children: React.ReactNode;
 }
 
+/**
+ * This component is a container component that has no semantic element. It provides a React Context
+ * model for all Popup subcomponents. If you manually pass a `model` to all subcomponents, this
```

Comment:
Is passing a `model` manually to all subcomponents practical? I wonder if we're over-documenting here.

Reply:
I realize this was relocated from somewhere else.

Reply:
Yes, this could be over-documenting. Here's an example of a nested popup with a manually passed model in the case where model contexts are nested and an override is desired: https://workday.github.io/canvas-kit/?path=/docs/components-popups-popup--nested-popups